src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -149,6 +149,7 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
         detection_time=ensure_aware(datetime.fromtimestamp(event["timestamp"])),
         culprit="user",  # TODO: fill in culprit correctly -- URL or paramaterized route/tx name?
         level=event.get("level", "info"),
+        initial_issue_priority=FeedbackGroup.default_priority,
```

Comment:
would it be possible to have `initial_issue_priority` be optional and use the default priority of the `type` parameter passed instead?

Reply:
That should be what we're doing already, `initial_issue_priority` is optional. I'm okay removing this line if it's cleaner to always use the default for Feedback issues.

Reply:
got it. yeah i think this makes sense.